### PR TITLE
fix(core): fixed card layout issues, growing card feature

### PR DESCRIPTION
**Pull request by platon-rov**
## Related Issue(s)

Relates to DXP.

## Description

Fixed Card Layout component 
* edge use cases fixes, 
* code optimisations, 
* growing cards processing feature.

### Before:
image

### After:
image

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html [hunk 1]
```diff
@@ -11,24 +11,18 @@
 </fd-segmented-button>
 
 <div style="padding: 1rem; overflow: auto">
-    <fd-fixed-card-layout (cardDraggedDropped)="cardDraggedDropped($event)">
+    <fd-fixed-card-layout (cardDraggedDropped)="cardDraggedDropped($event)" [disableDragDrop]="true">
```
Comment (mikerodonnell89) [inline apps/docs/src/app/core/component-docs/fixed-card-layout/examples/default/fixed-card-layout-examples.component.html:14]: Does it still need `(cardDraggedDropped)="cardDraggedDropped($event)"` if `[disableDragDrop]="true"`?

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
The "show/hide" buttons in the example are no longer working - guessing a result of changes to lifecycle methods

**Comment by g-cheishvili** [comment]
> 

It does same for me too

File: libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts [hunk 2]
```diff
@@ -23,15 +22,15 @@ import {
...
+import { CdkDrag, CdkDragDrop, CdkDragEnter } from '@angular/cdk/drag-drop';
+import { Subject, Subscription } from 'rxjs';
+import { debounceTime, delay, distinct, takeUntil } from 'rxjs/operators';
 
 import { resizeObservable, RtlService } from '@fundamental-ngx/core/utils';
 import { FixedCardLayoutItemComponent } from './fixed-card-layout-item/fixed-card-layout-item.component';
 import { coerceNumberProperty, NumberInput } from '@angular/cdk/coercion';
 
-const PX_IN_REM = 16;
+const REM_IN_PX = 16;
```
Comment (g-cheishvili) [inline libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts:33]: I we want to be more precise about it, we should get document's font-size computed value

**Review by g-cheishvili — COMMENTED** [review]
When I managed to resize it, it worked as expected. So I approve it, because goal is reached

**Comment by g-cheishvili** [comment]
Please fix merge conflicts

**Review by InnaAtanasova — APPROVED** [review]

**Review by g-cheishvili — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Pull request merged**